public/js/index.js

```diff
@@ -676,6 +657,21 @@ $("#board").on("click", (event) => {
 
          break;
 
+      case "learn-about-cards-btn":
+         $(".voteCard").each(function () { // must not be arrow function for scope
+            $(this).children('img').first().attr("id", "card-learn-button");
+            $(this).addClass("learn-btn-highlight");
+            $(this).append($('<button></button>'));
+         });
+         break;
+
+      case "card-learn-button":
+         const cardPosition = event.target.parentElement.id.split("-")[1];
+         let cardIndex = GameObject.roundData.cardArray[cardPosition].cardIdentifier
+         alert("Show Story of card " + cardIndex);
```

Comment:
We should avoid using the default browser alert. There's a static function that displays messages on the game board that could be used for these alerts. Or mabye even the chat.